src/components/ChatLog.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
```

Comment:
Looks like you never ended up adding the `PropTypes` for this file. While it's definitely a bit redundant with the `PropTypes` in `ChatEntry`, it's still good to use features like `PropTypes` to help catch errors early.

And if you _did_ decide to not use `PropTypes`, then at least remove the `import` statement just as part of keeping the code clean.